yarn.lock

```diff
@@ -1478,27 +1426,12 @@ fsevents@^1.2.3:
   dependencies:
     bindings "^1.5.0"
     nan "^2.12.1"
-    node-pre-gyp "*"
```

Comment:
We should fix this line just like what we did in https://github.com/ikatyang/angular-estree-parser/pull/151#discussion_r373771788.

Reply:
I added it back, and resolve the conflicts.

I checked the deps of `fsevents@1.2.11`, seems `node-pre-gyp` is not listed in `dependencies` but in `bundledDependencies`.

https://unpkg.com/browse/fsevents@1.2.11/package.json